### ISSUE TITLE
[Components and pattern standards] The components of the navigation system (i.e. URL, breadcrumb, left nav) within your content or feature don't accurately and/or consistently represent the location of the page within the IA. (04.01.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#30429]
#### VFS team name
Public Websites

#### VFS product name
ORDMI Harassment Reporting Tool - Landing Page

#### Point of Contact/Reviewers
Josh Kim (Accessibility)

---

### Platform Issue
The components of the navigation system (i.e. URL, breadcrumb, left nav) within your content or feature don't accurately and/or consistently represent the location of the page within the IA.

### Issue Details
Breadcrumb's `aria-current` is set to "Home" which links to "VA.gov" on other pages.

### VA.gov Experience Standard
[Cateogy Number 04, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.4 A

---

### Platform Recommendation
Breadcrumb should have an additional level that matches the `h1` on the page e.g. "How to report harassment at VA." `Aria-current` attribute should be set to this level.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by mnorthuis** [comment]
@joshkimux This will need to be validated post-launch.  The page has to be assigned to a "menu item" in Drupal for the breadcrumb to correctly build, and that can't be done until the page is published.   Not launch-blocking.

**Comment by laflannery** [comment]
Confirmed that the page has been published and `aria-current` has been set to the H1 breadcrumb:
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/106678594/b9d6fe6e-7174-4acb-82d0-055dd9e8fa2e)
